app/services/NuanceEncryptionService.scala

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
The Nuance Encryption service is in the incorrect place. It should be in uk.gov.hmrc.tai.services folder